src/diffpy/labpdfproc/tests/test_functions.py

```diff
@@ -56,27 +56,29 @@ def test_set_muls_at_angle(inputs, expected):
     assert actual_muls_sorted == pytest.approx(expected_muls_sorted, rel=1e-4, abs=1e-6)
 
 
-def test_compute_cve(mocker):
+def _instantiate_test_do(mocker, yarray, name="test", scat_quantity="x-ray"):
```

Comment:
if you pass everything in we don't gain much.    Are there cases where you want to vary name="test", scat_quantity="x-ray" etc?

I would make yarray a required arg too to make the code more readable.

Also, is it necessary to pass in mocker?  It may be, but if not I don't think it adds much syntatically.

Reply:
Yeah I couldn't find prettier numbers for the yarrays.. they all seem a bit messy.
The name and scat_quantity are for the absorption correction DOs.
I can remove the mocker, it's not needed there.